src/content/concepts/principles-of-ax.md

```diff
@@ -0,0 +1,106 @@
+
+**Current available solutions**
+
+Often, the “User-Agent” header will include agent information that can be helpful in identifying that a request is made by an agents.
+
+When providing a means to authenticate, ensure that the session information for the authentication is flagged as being an agent-based session.
+
+While digital services can’t control the information available in requests that agents send to their services they can influence it. Within the context instructions provided for how to interact with the digital service, instructions can be included to send metadata such as a custom header that says the request originated from an agent.
+
+
+
+---
+---
+
+## AX principles for agents
+
+While there is a lot for digital services to consider, agents have equally important considerations that will help improve the value and impact of agents that meet these principles.
+
+### Mindful Digital Collaboration
+
+Agents, as a persona of the digital ecosystem, should maintain a good standing with digital services by respecting the boundaries of digital services. Digital services define how they allow their site and systems to be used and, where desired, will block usage of certain mediums for their services. Their decision to do this can be for many reasons from hard costs to support burden to philosophical alignment of practices. **Agents should attempt to use the optimal practices and then any available route within the boundaries set by the digital service to achieve the goals requested by the end user.** When an agent can not complete its task within digital service boundaries, it should look to use alternative digital services or inform the end user where alternatives are explicitly not allowed. Digital services blocking agent usage across all access points or not providing sufficient AX should expect that agents will send end users to alternatives and competitors. If an agent is operating outside of the boundaries set by digital services, the agent should expect that its actions will be blocked or become unreliable.
+
+### Adopt optimal practices
+
+By their nature, agents consume digital services differently than end users using other direct interaction mediums like browsers and mobile apps. The difference often creates substantial costs as digital services have not been designed for or optimized for this nature in the past. Practices are improving for how digital services support agents best. Digital services supporting the latest AX patterns means they will be delivering context and interactivity in ways that are optimized for the consumption model of agents. Where optimal patterns are available, agents should use these to solve the customer need but also reduce the impact agent consumption has on digital services.
+
+### Agent Interactivity Patterns
```

Comment:
This entire section is a duplicate from the one at the top?

Reply:
kind of, it's calling out this principle for agents to achieve. It's reoriented for agents to support these and expect services to support them.